doc/source/whatsnew/v0.19.0.txt

```diff
@@ -229,6 +229,7 @@ Other enhancements
 - ``pd.read_html()`` has gained support for the ``decimal`` option (:issue:`12907`)
 - A top-level function :func:`union_categorical` has been added for combining categoricals, see :ref:`Unioning Categoricals<categorical.union>` (:issue:`13361`)
 - ``Series`` has gained the properties ``.is_monotonic``, ``.is_monotonic_increasing``, ``.is_monotonic_decreasing``, similar to ``Index`` (:issue:`13336`)
+- ``DataFrame.sort_values()`` has gained `axis=1` support to re-order columns with `by=index_label` (:issue:`10806`)
```

Comment:
use double-backquotes around axis=1 and ``by=index_label`` (though I am not entirely sure what that means). you can say ``by=`` I think the ``index_label`` is confusing.